rts/Game/Game.cpp

```diff
@@ -464,6 +464,7 @@ void CGame::Load(const std::string& mapFileName)
 			}
 			LEAVE_SYNCED_CODE();
 		}
```

Comment:
the "why" was about why the ENTER/LEAVE SYNCED CODE block was added, not about the actual contents of the block

Reply:
Is the code not supposed to be in a synced block? The code in this block was originally in such a block, it was moved out of the conditional because it makes sense to run it regardless of whether the match is start from new or a saved game.

Reply:
Sorry, I was confused. Looks good